src/ds/d500/d500-mapping.cpp

```diff
@@ -0,0 +1,247 @@
+// License: Apache 2.0. See LICENSE file in root directory.
+// Copyright(c) 2023 Intel Corporation. All Rights Reserved.
+
+#include "d500-mapping.h"
+
+#include <vector>
+#include <map>
+#include <cstddef>
+
+#include "ds/ds-timestamp.h"
+#include "ds/ds-options.h"
+#include "stream.h"
+
+namespace librealsense
+{
+    
+    const std::map<uint32_t, rs2_format> mapping_fourcc_to_rs2_format = {
+        {rs_fourcc('G','R','E','Y'), RS2_FORMAT_OCCUPANCY},
+        //point cloud - needed a standard format, with 32 bit per pixel:
+        // #define V4L2_PIX_FMT_ARGB32  v4l2_fourcc('B', 'A', '2', '4') /* 32  ARGB-8-8-8-8  */
```

Comment:
Why do we need this comment? `#define V4L2_PIX_FMT_ARGB32`

Reply:
This is the line from uvcvideo2.h header, aims to explain what is this 'B','A','2','4' format.
I ll add "from uvcvideo2.h linux standard file:"